Doc/library/runpy.rst

```diff
@@ -106,15 +106,16 @@ The :mod:`runpy` module provides two functions:
    Execute the code at the named filesystem location and return the resulting
    module globals dictionary. As with a script name supplied to the CPython
    command line, the supplied path may refer to a Python source file, a
-   compiled bytecode file or a valid sys.path entry containing a ``__main__``
-   module (e.g. a zipfile containing a top-level ``__main__.py`` file).
+   compiled bytecode file or a valid :data:`sys.path` entry containing a
+   ``__main__`` module
```

Comment:
```suggestion
   :mod:`__main__` module
```
?

Reply:
Would that link to anywhere in the Python docs? If not, better to leave it as it is, I think :)

Reply:
Yes, that would link to https://docs.python.org/3/library/__main__.html.

Reply:
Then go for it!

Reply:
Done 👍